examples/iam-auth.js

```diff
@@ -0,0 +1,50 @@
+/* eslint no-console:0 */
+//
+// Create an API client for an AWS EKS cluster using IAM authentication
+// Note: In order for this to work, you must set the environment
+// variable AWS_PROFILE
```

Comment:
`AWS_PROFILE` is used here: https://github.com/godaddy/kubernetes-client/pull/332/files#diff-7b7c8857ce22c1739c6ad1ea186ca9ddR22

Should this comment be in that file?